bf: ZENKO-2729 remove .only from raw-node tests

dora-korpar opened the pull request:
Remove .only in raw-node headObject tests

dora-korpar committed:
bf: ZENKO-2729 remove .only from raw-node tests

bert-e commented:
# Hello dora-korpar,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/cloudserver/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers





Peer approvals *must* include a mandatory approval from @dora-korpar.

rahulreddy reviewed (commented):
It's in 7.4 as well, can you please rebase against 7.4?

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/ZENKO-2729-raw-node-tests-remove-only` into
`development/7.4`
* [w/7.8/bugfix/ZENKO-2729-raw-node-tests-remove-only](https://github.com/scality/cloudserver/compare/development/7.8...w/7.8/bugfix/ZENKO-2729-raw-node-tests-remove-only) will be merged into `development/7.8` (pull request #2938)
* [w/8.1/bugfix/ZENKO-2729-raw-node-tests-remove-only](https://github.com/scality/cloudserver/compare/development/8.1...w/8.1/bugfix/ZENKO-2729-raw-node-tests-remove-only) will be merged into `development/8.1` (pull request #2939)
* [w/8.2/bugfix/ZENKO-2729-raw-node-tests-remove-only](https://github.com/scality/cloudserver/compare/development/8.2...w/8.2/bugfix/ZENKO-2729-raw-node-tests-remove-only) will be merged into `development/8.2` (pull request #2940)



The following branches will **NOT** be impacted:

* `development/6.4`




*Follow* integration pull requests if you would like to be notified of
build statuses by email.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers





Peer approvals *must* include a mandatory approval from @dora-korpar.

jonathan-gramain commented:
/bypass_author_approval

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/cloudserver/commit/abbe0e3c751a14c4aa99c72345b2ac59f2ef8a28) did not succeed in branch w/7.8/bugfix/ZENKO-2729-raw-node-tests-remove-only.




*The following options are set:* **bypass_author_approval**

jonathan-gramain committed:
bugfix: ZENKO-2729 async raw-node headObject test before/after

try to reduce flakiness by waiting until the before/after hooks are
done creating/deleting the test bucket

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/cloudserver/commit/1b7a0182c2a8a2a0cadd6918712e27fa753e2155) did not succeed in branch w/8.2/bugfix/ZENKO-2729-raw-node-tests-remove-only.




*The following options are set:* **bypass_author_approval**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/cloudserver/commit/c35cac674be3d7b588f27d9f48d845d0c7faaae2) did not succeed in branch w/8.1/bugfix/ZENKO-2729-raw-node-tests-remove-only.




*The following options are set:* **bypass_author_approval**

jonathan-gramain committed:
bugfix: ZENKO-2729 fix routeBackbeat mixed use of Promise and callback

In recent versions of mocha, the current test code before() hook in
routeBackbeat.js triggers an error "Error: Resolution method is
overspecified. Specify a callback *or* return a Promise; not both".

Fix this by not returning the promise object, so that more recent
versions of mocha are happy.

bert-e commented:
# Conflict




A conflict has been raised during the update of
integration branch `w/8.1/bugfix/ZENKO-2729-raw-node-tests-remove-only` with contents from `w/7.8/bugfix/ZENKO-2729-raw-node-tests-remove-only`
and `development/8.1`.

Please resolve the conflict on **the integration branch** (`w/8.1/bugfix/ZENKO-2729-raw-node-tests-remove-only`).


Here are the steps to resolve this conflict:

```bash
 $ git fetch
 $ git checkout w/8.1/bugfix/ZENKO-2729-raw-node-tests-remove-only
 $ git pull  # or "git reset --hard origin/w/8.1/bugfix/ZENKO-2729-raw-node-tests-remove-only"
 $ git merge origin/development/8.1
 $ # <intense conflict resolution>
 $ git commit
 $ git merge origin/w/7.8/bugfix/ZENKO-2729-raw-node-tests-remove-only
 $ # <intense conflict resolution>
 $ git commit
 $ git push -u origin w/8.1/bugfix/ZENKO-2729-raw-node-tests-remove-only
```





*The following options are set:* **bypass_author_approval**

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/7.4`

* :heavy_check_mark: `development/7.8`

* :heavy_check_mark: `development/8.1`

* :heavy_check_mark: `development/8.2`



The following branches will **NOT** be impacted:

* `development/6.4`
* `stabilization/7.4.7`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **bypass_author_approval**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/7.4`

* :heavy_check_mark: `development/7.8`

* :heavy_check_mark: `development/8.1`

* :heavy_check_mark: `development/8.2`



The following branches have **NOT** changed:

* `development/6.4`
* `stabilization/7.4.7`



Please check the status of the associated issue ZENKO-2729.

Goodbye dora-korpar.
